util/argo/resource_tracking.go

```diff
@@ -180,8 +180,10 @@ func (rt *resourceTracking) ParseAppInstanceValue(value string) (*AppInstanceVal
 	var appInstanceValue AppInstanceValue
 	parts := strings.Split(value, ":")
 	appInstanceValue.ApplicationName = parts[0]
-	if len(parts) != 3 {
+	if len(parts) < 2 {
 		return nil, WrongResourceTrackingFormat
+	} else if len(parts) > 3 {
+		parts = []string{parts[0], parts[1], strings.Join(parts[2:], ":")}
```

Comment:
I think it would be better to use `strings.SplitN(value, ":", 3)`, because the `<namespace>/<name>` potion (where an additional colon could appear) is at the end. 

This way, we would not have to introduce the additional `else` block and join back the resource name.

Reply:
Switched to using splitN